saito-rust/configs/saito.config.template.json

```diff
@@ -16,5 +16,10 @@
       "protocol": "http",
       "synctype": "full"
     }
-  ]
+  ],
+  "spammer": {
```

Comment:
this should be removed

Reply:
Okay will remove this in a future commit, adding as a TODO